exampleCourse/questions/element/hiddenHints/question.html

```diff
@@ -1,9 +1,12 @@
 <pl-question-panel>
   <p>
     Questions below were designed to showcase the different features of
-    <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-hidden-hints-element"><code>pl-hidden-hints</code></a>.
-    The goal of <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-hidden-hints-element"><code>pl-hidden-hints</code></a>
-    is to progressively show hints as students submit incorrect (but not invalid) answers. Submit incorrect answers to see more hints.
+    <a
+      href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-hidden-hints-element"><code>pl-hidden-hints</code></a>.
+    The goal of <a
+      href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-hidden-hints-element"><code>pl-hidden-hints</code></a>
+    is to progressively show hints as students submit incorrect (but not invalid) answers. Submit incorrect answers to
+    see more hints. Try 10 to get a special hint.
```

Comment:
Very much a nit, but since this is a special case not expected to be used often, I'd suggest moving the added comment to the end of the statement.